rancher2/structure_machine_config_v2_amazonec2.go

```diff
@@ -51,6 +51,25 @@ type machineConfigV2Amazonec2 struct {
 	VolumeType              string   `json:"volumeType,omitempty" yaml:"volumeType,omitempty"`
 	VpcID                   string   `json:"vpcId,omitempty" yaml:"vpcId,omitempty"`
 	Zone                    string   `json:"zone,omitempty" yaml:"zone,omitempty"`
+
+	// Enables or disables the IPv6 endpoint for the instance metadata service.
```

Comment:
non-blocking nit: 
```suggestion
	// HttpProtocolIpv6 Enables or disables the IPv6 endpoint for the instance metadata service.
```
Same for other fields as well, but again, non-blocking

Reply:
Fixed.
Thank you for reviewing.